handlers/APIHandler.js

```diff
@@ -0,0 +1,77 @@
+var db = require('../db/users.js');
+var User = require('../public/javascripts/userModel.js');
+var Transaction = require('../public/javascripts/transactionModel.js');
+var brcypt = require('bcrypt-nodejs');
+
+function APIHandler() {
+
```

Comment:
I read through the article and changed how I was sending some responses. Thanks @tarebyte!